src/components/Embeds/AirMap.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CITIES_LOCATION = {
+  nairobi: '12/-1.2709/36.8169',
+  lagos: '12/6.4552/3.4198',
+  'dar-es-salaam': '12/-6.8555/39.1518'
+};
+
+function AirMap({ location }) {
```

Comment:
QQ The `SensorMap` itself is already an `iframe`, why don't we return just that?

Reply:
sure, i don't have much resistance for that, i just found it weird to have iframe and another iframe.

Reply:
I meant lets return the _inner_ `iframe` itself. It'll be weird having two iframes!